data/transactions/logic/assembler_test.go

```diff
@@ -1492,10 +1432,7 @@ len
 	require.NoError(t, err)
 	require.Equal(t, ops2.Program, ops.Program)
 
-	_, err = AssembleString("#pragma unk")
-	require.Error(t, err)
-	require.Contains(t, err.Error(), "1: unsupported pragma directive: unk")
-
+	testProg(t, "#pragma unk", assemblerNoVersion, expect{1, "unsupported..."})
```

Comment:
*unsupported pragma*, `unsupported...` does not distinguish *unsupported version* and *unsupported pragma*